x-pack/solutions/observability/plugins/inventory/public/components/entity_actions/index.tsx

```diff
@@ -24,32 +24,28 @@ export const EntityActions = ({ entity, setShowActions }: Props) => {
     ? `inventoryEntityActionsButton-${entity.entityDisplayName}`
     : 'inventoryEntityActionsButton';
 
-  const { getDiscoverEntitiesRedirectUrl, isEntityDefinitionLoading } = useDiscoverRedirect(entity);
+  const { getDiscoverEntitiesRedirectUrl } = useDiscoverRedirect(entity);
```

Comment:
I noticed that the ad hoc data view is created for each row causing redundant requests 

https://github.com/user-attachments/assets/8410cf65-e20e-4c36-a4dc-7eb1e7df8a6d

Could you please take a look